atm.js

```diff
@@ -1,3 +1,111 @@
+// - Each additional deposit will add amount into 'balance' or total for each savings and checking.
+// - Conversely, When the user enters an amount and clicks 'Withdraw', the specified amount will be subtracted from the 'balance' or total.
+// - If possible, convert user input to two decimal places to calculate proper currency value and display.
+// - The user cannot withdraw more than the balance of each account ((wishlist: possible //overdraft protection, where if funds are available on either account to cover the full withdrawal of a potential 'overdraft', the needed amount is withdrawn)).
+// - When either account balance reaches 0, red warning color will show in background of text/balance display.
+
+
+
 $(document).ready(function(){
-// Uh oh -- it's saying `$` is undefined! Something's missing from `index.html`...
-});
+  console.log("I'm So Ready!");
+  $(".balance").addClass("zero");
+  //Checking
+  //click listener for function to add user input>
+  $("#checking .deposit").on("click", updatedCheckingDeposit);
+  //function for checking deposit
+  function updatedCheckingDeposit (){
+    //gets user numeric input from checking account deposit>
+    var depositChecking = Number.parseFloat($("#checking input").val());
+    console.log(depositChecking);
+    //separating the "$" from the "0">
+    var balanceTest1 = Number.parseFloat($("#checking .balance").text().split("$")[1]);
+    console.log(balanceTest1);
+    //adding
+    var balanceTest2 = Number.parseFloat(depositChecking) + balanceTest1;
+    console.log(balanceTest2);
+    //continuing concatenating and placing value into balance field
+    var checkingBalance = ("$") + (balanceTest2).toFixed(2);
+    $("#checking .balance").html(checkingBalance);
+    console.log(checkingBalance);
+    $("#checking .input").val("");
+    if (balanceTest2 > 0) {
+      $("#checking .balance").removeClass("zero");
+    }
+  }
+  //click listener for function to add user input>
+  $("#checking .withdraw").on("click", updatedWithdrawal);
+  //function for checking withdrawal
+  function updatedWithdrawal (){
+    //gets user numeric input from checking account deposit>
+    var withdrawChecking = Number.parseFloat($("#checking input").val());
+    console.log(withdrawChecking);
+    //separating the "$" from the "0">
+    var balanceTest1 = Number.parseFloat($("#checking .balance").text().split("$")[1]);
+    console.log(balanceTest1);
+    //subtracting
+    var balanceTest2 = balanceTest1 - Number.parseFloat(withdrawChecking);
+    console.log(balanceTest2);
+    //concatenating and placing value into balance field
+    //var checkingBalance = ("$") + (balanceTest2).toFixed(2);
+    //$("#checking .balance").html(checkingBalance);
+    //console.log(checkingBalance);
+    $("#checking .input").val("");
+    if (balanceTest2 < 0) {
+      alert ("whoa");
+    }
+    else {
+      $("account .balance").html(("$") + (balanceTest2).toFixed(2))
+    }
+  }
```

Comment:
For both checking and saving, what you need to do is capture the val() from checking or savings input (the class) and subtract from balance using the html() method. Store this to a variable--maybe called total-- and then use this to set the new inner html of balance.